en-US/S.xml

```diff
@@ -112,16 +112,21 @@
         </table>
       </listitem>
     </varlistentry>
-    <!-- 		 <varlistentry id="see">
+    	 <varlistentry id="see">
 			<term>see</term>
 			 <listitem>
 				<para>
-					Use to refer readers to another resource. Avoid using "refer to" in this context.
+				For references within a document, and for cross-references to other resources, do not use “see”. Use “refer to”.
+				</para>
+				<para>
+				For descriptions of graphics, explanations of command outputs, and other similar cases, it is acceptable to use “see”. 
+				Alternatively, it might be more concise or precise to replace “see” with another verb in these cases.
+				For example, instead of using "Click a server link to see the result details", you could use "Click a server link for the result details. 
```

Comment:
```suggestion
				For example, instead of using "Click a server link to see the result details", you could use "Click a server link for the result details". 
```